src/experimental/effects/reflection-effect/reflection-effect.js

```diff
@@ -35,7 +41,7 @@ export default class ReflectionEffect extends Effect {
         vertices: new Float32Array([0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0])
```

Comment:
Perhaps `this.model` could have a more specific name here? Like this.unitQuad or something?

Layers have some built-in assumptions about `model` not sure that effects need to.